docs/fundamentals/networking/sockets/sockets-overview.md

```diff
@@ -23,7 +23,7 @@ helpviewer_keywords:
 
 The <xref:System.Net.Sockets> namespace contains a managed, cross-platform socket networking implementation. All other network-access classes in the <xref:System.Net> namespace are built on top of this implementation of sockets.
 
-The .NET Framework <xref:System.Net.Sockets.Socket> class is a managed-code version of the socket services provided by the Winsock32 API. In most cases, the `Socket` class methods simply marshal data into their native Win32 counterparts and handle any necessary security checks.
+The <xref:System.Net.Sockets.Socket> class is a managed-code version of the socket services provided relying on native interoperability with either Linux or Windows. In most cases, the `Socket` class methods simply marshal data into their native counterparts and handle any necessary security checks.
```

Comment:
We should call out MacOS specifically. (It's a unix OS, but it's not a Linux distro):

Reply:
Great callout, I was a bit confused by that myself - so yes. The two classes are `Socket.Unix` and `Socket.Windows` in the source code.

Reply:
Thanks. I'll resolve and approve.